lib/dokumi/tool/xcode/project_helper.rb

```diff
@@ -108,6 +108,18 @@ def update_provisioning_profiles(provisioning_profiles)
           @xcodeproj.save
         end
 
+        # Set the provisioning profile specifiers for multiple targets. The key is the target's' bundle identifier.
+        def update_provisioning_profile_specifiers(provisioning_profile_specifiers)
+          each_build_settings do |build_settings|
+            bundle_identifier = build_settings["PRODUCT_BUNDLE_IDENTIFIER"]
+            next unless bundle_identifier
+            provisioning_profile_specifier = provisioning_profile_specifiers[bundle_identifier]
+            next unless provisioning_profile_specifier
+            build_settings["PROVISIONING_PROFILE_SPECIFIER"] = provisioning_profile_specifier
```

Comment:
You should also remove existing provisioning profile specifiers as done in `update_provisioning_profiles`.
```ruby
build_settings.keys.each do |setting_name|
  build_settings.delete(setting_name) if setting_name.start_with?("PROVISIONING_PROFILE_SPECIFIER[sdk=")
end
build_settings["PROVISIONING_PROFILE_SPECIFIER"] = provisioning_profile_specifier
```